tests/test_views_rest.py

```diff
@@ -35,7 +35,148 @@
 from flask_security import url_for_security
 from six import BytesIO
 
-from invenio_deposit.api import Deposit
+
+def test_edit_deposit_1(app, db, es, users, location, deposit, write_token):
```

Comment:
I don't think is a good practice simply "write as many words you can" but briefly describe what you are doing.
If you look, the function title is not like "test_1, test_2, ..." but "test_edit_deposit_X".
It gives you enough information to understand the context: 
 - `test`: you are testing
 - `edit`: the action to modify
 - `deposit`: the object of your test
So, in the end, you are quickly aware that you are testing the editing of a deposit.
The number are there to say that all them are parts of "testing the editing of a deposit", but in different way: one time with correct permission, another time as anonymous user, another time with oauth2 token, ...
It give you a further information: leave them together because, for security reasons, the editing of a deposit is not fully tested without one of them.